sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -0,0 +1,25 @@
+package com.vimeo.dummy.sample_kotlin
+
+import com.google.gson.annotations.SerializedName
+import com.vimeo.stag.UseStag
+
+/**
+ * Test kotlin model
+ *
+ * Created by restainoa on 5/8/17.
+ */
+@UseStag
+class KotlinSamples {
+    var stringField: String? = null
+
+    var nonNullStringField: String = "default"
+
+    var intField: Int? = null
+
+    var longField: Long = 1
+
+    @SerializedName("boolean_field")
+    var booleanField: Boolean? = null
+
+    var testField: Any? = null
```

Comment:
For reference in this discussion: https://kotlinlang.org/docs/reference/visibility-modifiers.html

The way Kotlin member variables work is that they can only be accessed through getters and setters. Setters and getters are automatic and are left out of the source code unless you have different logic (see https://kotlinlang.org/docs/reference/properties.html). Looking at the corresponding generated Java code, when you specify public visibility on a field in Kotlin, you are giving public visibility to the getters and setters, but the field itself is always declared private. 

For json model objects, you could make your class immutable by declaring all the fields in the constructor, but gson functions by just assigning directly to fields rather than through a constructor, so stag functions the same (except using methods rather than fields in this case).

You can change the visibility of setters and getters individually (refer to the properties link above). Right now stag only supports public setters in kotlin. Maybe it should support `internal`, but this current code can't because `internal` behaves different from package local and therefore needs some special logic to access. Maybe we should file an issue about internal setters and getters for a future PR, or maybe it should be part of this PR.